src/raftstore/store/worker/read.rs

```diff
@@ -103,16 +103,17 @@ impl ReadDelegate {
                 {
                     // Cache snapshot_time for remaining requests in the same batch.
                     *last_valid_ts = snapshot_time;
-                    if let Ok(mut resp) = executor.execute(req, &self.region) {
-                        // Leader can read local if and only if it is in lease.
-                        cmd_resp::bind_term(&mut resp.response, term);
-                        return Some(resp);
-                    }
+                    let mut resp = executor.execute(req, &self.region);
```

Comment:
if `check_region_epoch()` failed in `execute()`, now `handle_read()` will return `Some(resp with snap is None)` and before will return None, is that alright?

Reply:
Yes, it is okay, because we disable epoch check in localreader.

Reply:
I think this will become a pitfall in the future.

Reply:
Could you make an example?

Reply:
It's not obvious for reader. If a reviewer misunderstood it, people could make the same mistake too when refactoring the code.